include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -211,9 +215,6 @@ struct SPhysicalDeviceLimits
     bool filterMinmaxSingleComponentFormats = false;
     bool filterMinmaxImageComponentMapping = false;
 
-    core::bitflag<asset::IImage::E_SAMPLE_COUNT_FLAGS> framebufferIntegerColorSampleCounts = asset::IImage::E_SAMPLE_COUNT_FLAGS(0u);
```

Comment:
one does not simply remove fields from this struct, one commens them out and gives a reason why

Reply:
nvm see you moved it